frontend/src/common/constansts/paths.ts

```diff
@@ -1,10 +1,11 @@
 export enum Paths {
-    Profile = "/profile",
-    TopUp = "/topup",
-    Transactions = "/transactions",
-    Home = "/home",
-    SignIn = "/signin",
-    SignUp = "/signup"
+    PROFILE = "/profile",
+    TOP_UP = "/topup",
+    TRANSACTIONS = "/transactions",
+    HOME = "/home",
```

Comment:
actually, your home page should be without "/home" just default route `localhost:3000`